hands_on/local_maxima_part2/test_local_maxima.py

```diff
@@ -23,8 +24,18 @@ def test_find_maxima_empty():
 
 
 def test_find_maxima_plateau():
-    raise Exception('not yet implemented')
+    values = [1, 2, 2, 1]
+    expected_v1 = [1]
+    expected_v2 = [2]
+    expected_v3 = [1, 2]
+
+    assert (np.all(find_maxima(values) == expected_v1) 
+            or np.all(find_maxima(values) == expected_v2) 
+            or np.all(find_maxima(values) == expected_v3))
```

Comment:
Nice, two comments:
1) find_maxima returns a list, so there's no need to use `np.all`: two list are equal if all their elements are equal
2) in a real-life situation, you would make a choice between one of these possible options and test that one specifically